src/main/java/resources/base.java

```diff
@@ -23,8 +23,7 @@ public WebDriver initializeDriver() throws IOException
 {
 	
  prop= new Properties();
-String currentDirectory = System.getProperty("user.dir");
-FileInputStream fis=new FileInputStream(currentDirectory+"\\src\\main\\java\\resources\\data.properties");
+FileInputStream fis=new FileInputStream(System.getProperty("user.dir")+"\\src\\main\\java\\resources\\data.properties");
```

Comment:
why you have created 2 new **fis** should't be a different name fis1 or something!!